### PR TITLE
Added correct language for code blocks

### PR DESCRIPTION
**Pull request by nainardev**
Ready to merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,24 +47,28 @@ Upcoming features:
 - dynamic size
 
 ## How to install
-
-    npm i heapify
+```sh
```
Comment (luciopaiva) [inline README.md:50]:
```suggestion

```sh
```

File: README.md [hunk 2]
```diff
@@ -47,24 +47,28 @@ Upcoming features:
...
+npm i heapify
+```
 
 Or if you're a yarn person:
-
-    yarn add heapify
+```sh
```
Comment (luciopaiva) [inline README.md:55]:
```suggestion

```sh
```

File: README.md [hunk 3]
```diff
@@ -47,24 +47,28 @@ Upcoming features:
...
+yarn add heapify
+```
 
 If you're on a browser, there's also the option of using a CDN:
-
-    import Heapify from "https://unpkg.com/heapify";
+```js
```
Comment (luciopaiva) [inline README.md:60]:
```suggestion

```js
```

File: README.md [hunk 4]
```diff
@@ -47,24 +47,28 @@ Upcoming features:
...
+import Heapify from "https://unpkg.com/heapify"
+```
 
 And to import a specific version:
-
-    import Heapify from "https://unpkg.com/heapify@0.2.1";
+```js
```
Comment (luciopaiva) [inline README.md:65]:
```suggestion

```js
```

File: README.md [hunk 5]
```diff
@@ -77,12 +81,14 @@ queue.pop();  // undefined
 ```
 
 ## Running tests
-
-    npm run test
+```sh
```
Comment (luciopaiva) [inline README.md:84]:
```suggestion

```sh
```

File: README.md [hunk 6]
```diff
@@ -77,12 +81,14 @@ queue.pop();  // undefined
...
+npm run test
+````
 
 For benchmark tests:
-
-    npm run bench
+```sh
```
Comment (luciopaiva) [inline README.md:89]:
```suggestion

```sh
```

File: README.md [hunk 7]
```diff
@@ -171,7 +177,7 @@ queue.peekPriority();  // 10
 
 Returns a string with an array representation of all priorities in the queue. For instance, given the following priority heap:
 
-```
+```sh
```
Comment (luciopaiva) [inline README.md:180]: This is just text, not a shell script

**Comment by nainardev** [comment]
Done!

**Pull request merged**